test/evergreen.yml

```diff
@@ -278,6 +278,27 @@ functions:
         set -o verbose
         ./t ${checkpoint_args} 2>&1
 
+  "checkpoint stress test":
+    command: shell.exec
+    params:
+      working_dir: "wiredtiger/test/format"
+      script: |
+        set -o errexit
+        set -o verbose
+
+        export WIREDTIGER_CONFIG='checkpoint_sync=0,transaction_sync=(method=none)'
+        CMD='./t -h WT_TEST.$i.$t -t r -r 2 -W 3 -n 1000000 -k 1000000 -C "cache_size=100MB"'
+
+        for i in $(seq ${times|1}); do
+          for t in $(seq ${no_of_procs|1}}); do
+            eval nohup $CMD > nohup.out.$i.$t 2>&1 &
+          done
+
+          for t in $(seq ${no_of_procs|1}); do
+            wait -n || exit $?
```

Comment:
I had a vague memory this line needs bash to run properly?

Reply:
This is actually to wait for the processes which was triggered with nohup command in the background.